imports/ui/components/comments/Comments.js

```diff
@@ -14,12 +14,20 @@ export default class Comments extends Component {
   getAuthorName(authorId) {
     return Person.findOne({userId: authorId});
   }
+  // render() {
+  //   const style = this.props.comments ? 'collection chat' : '111';
+  // }
+  componentDidMount() {
+    if (!this.props.comments.length - 1) {
```

Comment:
this.props.comments.length <= 1 is more readable
anyway, better to use classnames package for assigning css classes, otherwise on adding new comments this code will not rerun, because it should run only once on mount event